resources/constants.rb

```diff
@@ -143,7 +139,7 @@ def self.DuctSideSupply
     return 'supply'
   end
 
-  def self.DuctedInfoMiniSplitHeatPumpOrEvapCooler
+  def self.OptionallyDuctedSystemIsDucted
```

Comment:
If we kept getting more optionally ducted systems, we'd end up with DuctedInfoMiniSplitHeatPumpOrEvapCoolerOrAnotherSystemOrYetAnotherSystem 😆. I figure this is better.